dotnet/samples/Demos/FunctionInvocationApproval/Options/OpenAIOptions.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+namespace FunctionInvocationApproval.Options;
+
+/// <summary>
+/// Configuration for OpenAI chat completion service.
+/// </summary>
+public class OpenAIOptions
+{
+    public const string SectionName = "OpenAI";
+
+    /// <summary>
+    /// OpenAI model ID, see https://platform.openai.com/docs/models.
+    /// </summary>
+    public string ChatModelId { get; set; }
+
+    /// <summary>
+    /// OpenAI API key, see https://platform.openai.com/account/api-keys
+    /// </summary>
+    public string ApiKey { get; set; }
+
+    public bool IsValid =>
+        !string.IsNullOrWhiteSpace(this.ChatModelId) &&
+        !string.IsNullOrWhiteSpace(this.ApiKey);
+}
```

Comment:
Should this include OrgId?

Reply:
`IsValid` should not include `OrgId`, because this parameter is optional. Taking that into account, I decided to skip this parameter to keep the example simple.